### PR TITLE
Fixed route and implemented the videoid_int page

### PR DESCRIPTION
**Pull request by developingright**
Hi @razzeee,
I have fixed the routing for the videoid_int page, implemented its front-end, and backend. 

CPT2208081515-1832x972 (1)

### CODE REVIEW

**Comment by developingright** [comment]
Hi @razzeee,
The issue with the update function not working for the `videoid_int` page was because of its update function not being defined in `Pages.elm` file in the previous implementation of the code. So, I went ahead and fixed it.

**Comment by developingright** [comment]
Hi @razzeee,
I have implemented the video-playback backend for the `videoid_int` page and a modal window for the html5 video player. 
Please let me know if any changes are required.

File: public/main.css [hunk 1]
```diff
@@ -319,10 +319,55 @@ h2 {
...
+  ) !important;
+  mask-image: linear-gradient(
+    to right,
+    rgba(0, 0, 0, 1),
+    rgba(0, 0, 0, 0)
+  ) !important;
+}
+.video-modal {
+  z-index: 3;
+  width: 100%;
```
Comment (razzeee) [inline public/main.css:339]: try `100vw`

File: public/main.css [hunk 2]
```diff
@@ -319,10 +319,55 @@ h2 {
...
+  height: 100%;
```
Comment (razzeee) [inline public/main.css:340]: try `100vh`

File: src/Pages/Music/Videos/Videoid_Int.elm [hunk 3]
```diff
@@ -34,19 +47,37 @@ type alias Model =
...
-      , video = getVideos params.videoid shared.video_list
-      , album_list = shared.album_list
-      , artist_list = shared.artist_list
-      , video_list = shared.video_list
-      }
-    , Cmd.none
-    )
+    case getVideos params.videoid shared.video_list of
+        Nothing ->
+            ( { videoid = params.videoid
```
Comment (razzeee) [inline src/Pages/Music/Videos/Videoid_Int.elm:59]: You could move the shared object here into a variable via a `let ... in ...`

**Review by razzeee — CHANGES_REQUESTED** [review]
Just some minor things

**Comment by developingright** [comment]
Hi @razzeee, 
I have made the required changes.

File: src/Pages/Music/Videos/Videoid_Int.elm [hunk 4]
```diff
@@ -34,19 +47,30 @@ type alias Model =
...
-    ( { videoid = params.videoid
-      , video = getVideos params.videoid shared.video_list
-      , album_list = shared.album_list
-      , artist_list = shared.artist_list
-      , video_list = shared.video_list
-      }
-    , Cmd.none
-    )
+    let
+        sharedobjects =
```
Comment (razzeee) [inline src/Pages/Music/Videos/Videoid_Int.elm:58]: Not a fan of this naming, maybe `initialState`?

Comment (razzeee) [reply src/Pages/Music/Videos/Videoid_Int.elm:58]: Or better `initialModel`

Comment (developingright) [reply src/Pages/Music/Videos/Videoid_Int.elm]: Done :+1:

**Pull request merged**